### PR TITLE
Updates Cloud

### PR DESCRIPTION
**Pull request by bs-ovh**
incl. KFT0007525

### CODE REVIEW

File: pages/bare_metal_cloud/virtual_private_servers/upsize_vps_partition/guide.fr-fr.md [hunk 1]
```diff
@@ -1,68 +1,91 @@
...
-Lors de l’upgrade de votre VPS, il est possible qu’un repartitionnement de votre espace de stockage soit nécessaire.
+Après avoir augmenté la capacité de stockage de votre VPS, vous devrez repartitionner l'espace disque pour profiter de la taille réelle. Les étapes suivantes décrivent comment procéder.
 
 > [!warning]
 >
 > Le repartitionnement peut endommager définitivement vos données. OVHcloud ne pourra être tenu responsable de leur détérioration ou de leur perte. Avant de faire quoi que ce soit, pensez donc à bien sauvegarder vos informations. 
 >
 
-**Découvrez comment augmenter votre espace de stockage après un upgrade de votre VPS.**
+**Ce guide vous explique comment augmenter votre espace de stockage après une mise à niveau de disque.**
```
Comment (Kazarus4) [inline pages/bare_metal_cloud/virtual_private_servers/upsize_vps_partition/guide.fr-fr.md]:
```suggestion
**Découvrez comment augmenter votre espace de stockage après une mise à niveau de disque.**
```

File: pages/bare_metal_cloud/virtual_private_servers/upsize_vps_partition/guide.fr-fr.md [hunk 2]
```diff
@@ -1,68 +1,91 @@
...
 
-- Avoir un accès administrateur au VPS (Windows).
-- Avoir redémarré le serveur en [mode rescue](/pages/bare_metal_cloud/virtual_private_servers/rescue) (Linux).
+- Avoir un accès administrateur au VPS ([Windows](#windows)).
+- Avoir redémarré le serveur en [mode rescue](/pages/bare_metal_cloud/virtual_private_servers/rescue) (Linux uniquement).
 
 ## En pratique
 
-Suite à un upgrade, la mémoire vive (RAM) et le processeur (CPU) seront automatiquement ajustés. Cela ne sera pas nécessairement le cas pour l’espace de stockage.
+Après une mise à niveau de la mémoire (RAM) ou du processeur (vCores), ces ressources de votre VPS sont automatiquement ajustées. L'espace disque utilisable n'est cependant pas automatiquement augmenté lors de l'upgrade de stockage de votre VPS.
```
Comment (Kazarus4) [inline pages/bare_metal_cloud/virtual_private_servers/upsize_vps_partition/guide.fr-fr.md]:
```suggestion
Après une mise à niveau de la mémoire (RAM) ou du processeur (vCores), ces  deux ressources sont automatiquement ajustées sur votre VPS ; Contrairement à l'espace disque lors de l'upgrade de stockage de votre VPS.
```

File: pages/bare_metal_cloud/virtual_private_servers/upsize_vps_partition/guide.fr-fr.md [hunk 3]
```diff
@@ -1,68 +1,91 @@
...
 
 ### Linux
 
 #### Sauvegarder vos données
 
-La tentative d’étendre une partition peut entraîner une perte de données. Par conséquent, **nous vous recommandons vivement** de sauvegarder les informations de votre VPS.
+La tentative d’étendre une partition peut entraîner une perte de données. Il est donc **fortement recommandé** de faire une sauvegarde des données de votre VPS.
```
Comment (Kazarus4) [inline pages/bare_metal_cloud/virtual_private_servers/upsize_vps_partition/guide.fr-fr.md]:
```suggestion
La tentative d’étendre une partition peut entraîner une perte de données. Il est donc **vivement recommandé** de faire une sauvegarde des données de votre VPS.
```

File: pages/bare_metal_cloud/virtual_private_servers/upsize_vps_partition/guide.fr-fr.md [hunk 4]
```diff
@@ -1,68 +1,91 @@
...
 
-```sh
+Vous pourrez ensuite vérifier la configuration des disques :
+
+```bash
 lsblk
 ```
 
-La partition correspondant au mode rescue sera celle montée dans le répertoire `/`, qui est en réalité la racine du système. En revanche, la partition de votre VPS sera probablement placée dans un répertoire associé à « /mnt ».
+La partition correspondant au mode rescue (`sda1` dans cet exemple) est montée dans le répertoire `/` et le disque du VPS est nommé `sdb` et ne doit avoir aucun point de montage.
```
Comment (Kazarus4) [inline pages/bare_metal_cloud/virtual_private_servers/upsize_vps_partition/guide.fr-fr.md]:
```suggestion
La partition correspondant au mode rescue (`sda1` dans cet exemple) est montée dans le répertoire `/` .Quant à lui, le disque du VPS est nommé `sdb` et ne doit avoir aucun point de montage.
```

File: pages/bare_metal_cloud/virtual_private_servers/upsize_vps_partition/guide.fr-fr.md [hunk 5]
```diff
@@ -1,68 +1,91 @@
...
-```sh
+```console
 NAME MAJ:MIN RM SIZE RO TYPE MOUNTPOINT
 sda 254:0 0 10G 0 disk
 └─sda1 254:1 0 10G 0 part /
 sdb 254:16 0 25G 0 disk
+└─sdb1 254:17 0 25G 0 part
+```
+
+Si votre résultat ressemble à la sortie ci-dessus et que la colonne `MOUNTPOINT` est vide dans la ligne correspondante, vous pouvez passer à [l'étape suivante](#checkfs).
```
Comment (Kazarus4) [inline pages/bare_metal_cloud/virtual_private_servers/upsize_vps_partition/guide.fr-fr.md]:
```suggestion
Si votre résultat ressemble à l'exemple ci-dessus et que la colonne `MOUNTPOINT` est vide dans la ligne correspondante, vous pouvez passer à [l'étape suivante](#checkfs).
```

**Review by Kazarus4 — COMMENTED** [review]
FRFR proof done for me.

File: pages/bare_metal_cloud/virtual_private_servers/upsize_vps_partition/guide.fr-fr.md [hunk 6]
```diff
@@ -1,68 +1,91 @@
...
 
-- Avoir un accès administrateur au VPS (Windows).
-- Avoir redémarré le serveur en [mode rescue](/pages/bare_metal_cloud/virtual_private_servers/rescue) (Linux).
+- Avoir un accès administrateur au VPS ([Windows](#windows)).
+- Avoir redémarré le serveur en [mode rescue](/pages/bare_metal_cloud/virtual_private_servers/rescue) (Linux uniquement).
 
 ## En pratique
 
-Suite à un upgrade, la mémoire vive (RAM) et le processeur (CPU) seront automatiquement ajustés. Cela ne sera pas nécessairement le cas pour l’espace de stockage.
+Après une mise à niveau de la mémoire (RAM) ou du processeur (vCores), ces  deux ressources sont automatiquement ajustées sur votre VPS ; Contrairement à l'espace disque lors de l'upgrade de stockage de votre VPS.
```
Comment (OVH-Team-Guides) [inline pages/bare_metal_cloud/virtual_private_servers/upsize_vps_partition/guide.fr-fr.md]:
```suggestion
Après une mise à niveau de la mémoire (RAM) ou du processeur (vCores), ces  deux ressources sont automatiquement ajustées sur votre VPS, contrairement à l'espace disque lors de l'upgrade de stockage de votre VPS.
```

File: pages/bare_metal_cloud/virtual_private_servers/upsize_vps_partition/guide.fr-fr.md [hunk 7]
```diff
@@ -1,68 +1,91 @@
 ---
-title: 'Repartitionner un VPS suite à un upgrade'
-excerpt: 'Découvrez comment augmenter votre espace de stockage après un upgrade de votre VPS'
-updated: 2021-05-18
+title: Repartitionner un VPS après un upgrade de stockage
+excerpt: Découvrez comment augmenter l'espace disque utilisable suite à une mise à niveau
```
Comment (OVH-Team-Guides) [inline pages/bare_metal_cloud/virtual_private_servers/upsize_vps_partition/guide.fr-fr.md]:
```suggestion
excerpt: "Découvrez comment augmenter l'espace disque utilisable suite à une mise à niveau"
```

File: pages/bare_metal_cloud/virtual_private_servers/upsize_vps_partition/guide.fr-fr.md [hunk 8]
```diff
@@ -1,68 +1,91 @@
...
-Lors de l’upgrade de votre VPS, il est possible qu’un repartitionnement de votre espace de stockage soit nécessaire.
+Après avoir augmenté la capacité de stockage de votre VPS, vous devrez repartitionner l'espace disque pour profiter de la taille réelle. Les étapes suivantes décrivent comment procéder.
 
 > [!warning]
 >
 > Le repartitionnement peut endommager définitivement vos données. OVHcloud ne pourra être tenu responsable de leur détérioration ou de leur perte. Avant de faire quoi que ce soit, pensez donc à bien sauvegarder vos informations. 
 >
 
-**Découvrez comment augmenter votre espace de stockage après un upgrade de votre VPS.**
+**Découvrez comment augmenter votre espace de stockage après une mise à niveau de disque.**
```
Comment (OVH-Team-Guides) [inline pages/bare_metal_cloud/virtual_private_servers/upsize_vps_partition/guide.fr-fr.md]:
```suggestion
**Découvrez comment augmenter votre espace de stockage après un upgrade de disque.**
```

**Pull request merged**